backends/p4test/run-p4-sample.py

```diff
@@ -119,7 +119,8 @@ def target():
 timeout = 10 * 60
 
 def compare_files(options, produced, expected, ignore_case):
-    if options.replace:
+    # p4info files should not change
+    if options.replace and "p4info" not in produced:
```

Comment:
Is the check also implemented for the other back end `run-p4-sample.py` files?

Reply:
I think that only this script generates and compares the p4info files.
This is the only script that generates and compares the reference output files.

Reply:
Oh I see. So when you have files such as `xyz-bmv2.p4` tests are run twice. Once with p4test and the other time with the bmv2 script.